torch/_inductor/compile_fx.py

```diff
@@ -1454,7 +1461,10 @@ def codegen_and_compile(
                     with dynamo_timed(
                         "GraphLowering.compile_to_fn", log_pt2_compile_event=True
                     ):
-                        if graph.aot_mode:
+                        if graph.aot_mode and graph.fx_wrapper:
```

Comment:
Are `graph.cpp_wrapper` and `graph.fx_wrapper` mutually exclusive? Longer term, it might make sense to refactor them into the some sort of ABC.

```suggestion
                        if graph.aot_mode and graph.fx_wrapper:
                            assert not graph.cpp_wrapper
```

Reply:
Yup good point! I've refactored it so that fx_wrapper and cpp_wrapper are now exclusive.